### PR TITLE
feat: add LI.FI widget

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

We want to natively facilitate bridging in the UI. This adds the LI.FI widget, accessed via a new "Bridge" sidebar element.

## How this PR fixes it

A new sidebar element ("Bridge") opens a new `/bridge` route, displaying the LI.FI widget.

## How to test it

Open the "Bridge" section of a Safe and observe the widget, with the ability to bridge.

## Screenshots

image

image

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/sidebar/SidebarNavigation/config.tsx [hunk 1]
```diff
@@ -20,49 +21,58 @@ export type NavItem = {
...
+    {
+      label: 'Swap',
+      icon: <SvgIcon component={SwapIcon} inheritViewBox />,
+      href: AppRoutes.swap,
+    },
+    {
+      label: 'Stake',
+      icon: <SvgIcon component={StakeIcon} inheritViewBox />,
+      href: AppRoutes.stake,
+      tag: !isBridgeEnabled ? chip : undefined,
```
Comment (iamacook) [inline src/components/sidebar/SidebarNavigation/config.tsx:52]: We should confirm if we want to outright remove the chip from staking and, if so, we can remove the extra logic involved.

**Review by tmjssz — APPROVED** [review]

File: src/components/wrappers/FeatureWrapper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { Navigate } from '@/components/common/Navigate'
+import { useHasFeature } from '@/hooks/useChains'
+import madProps from '@/utils/mad-props'
+import type { FEATURES } from '@/utils/chains'
+
+// TODO: Use with swaps/staking
+export function _FeatureWrapper({
+  children,
+  feature,
+  to,
```
Comment (katspaugh) [inline src/components/wrappers/FeatureWrapper/index.tsx:12]: I would rename this prop to `fallbackRoute`.

**Review by katspaugh — APPROVED** [review]
Nice refactoring along with the feature itself!

File: src/components/wrappers/FeatureWrapper/index.tsx (hunk 2 above)
Comment (iamacook) [reply src/components/wrappers/FeatureWrapper/index.tsx]: Renamed in c5dc7e745121a5eefdb5a14ba92986d21b08d224.

**Pull request merged**